admin_guide/managing_networking.adoc

```diff
@@ -288,17 +288,20 @@ Or,
 - A pod cannot reach specified internal subnets/hosts that it should have no
 reason to contact.
 
-You can configure projects to have different egress policies. For example,
-allowing `<project A>` access to a specified IP range, but denying the same
-access to `<project B>`. Or restrict application developers from updating from
-(Python) pip mirrors, and forcing updates to only come from desired sources.
+Egress policies can be set at the pod selector-level and project-level. For
+example, allowing `<project A>` access to a specified IP range, but denying the
+same access to `<project B>`. Or restrict application developers from updating
```

Comment:
s/Or/You can also
no comma
s/forcing/force
s/desired/approved (or specified, or some other adjective. Both the CCS glossary and IBM Style say "no" to desire.)